purview/api/v1.py

```diff
@@ -105,26 +105,15 @@ def _get_partial_story(query, resources_to_expand):
         results_unordered.update(
             _get_partial_story(backward_query, [resource]))
 
-    results = OrderedDict({'data': [], 'meta': {}})
-    results['meta']['related_nodes'] = {key: 0 for key in story_flow.keys()}
-
     # Adding the artifact itself if it's story is not available
     if len(results_unordered) == 0:
+        results = {'data': [], 'meta': {}}
```

Comment:
Optional: You could call `_order_story_results` instead of these two lines.

Reply:
That would just become confusing, hence I would like it to be as it is. :)